lib/PAUSE/Indexer/Errors.pm

```diff
@@ -124,30 +141,48 @@ EOF
 };
 
 pkg_error dual_newer => {
-  # TODO bring back parameters
   header  => 'Not indexed because of an newer dual-life module',
-  body    => <<'EOF',
-Not indexed because package $opack in file $ofile has a dual life in $odist.
-The other version is at $oldversion, so not indexing seems okay.
+  body    => sub {
+    my ($old) = @_;
+
+    _assert_args_present(db_conflict => $old, [ qw(package file dist version) ]);
+
+    return <<"EOF";
+Not indexed because package $old->{pack} in file $old->{file} has a dual life
```

Comment:
You expect 'package' but you're using 'pack' so this throws a warning and doesn't fill in